src/plugins/data/server/index_patterns/fetcher/lib/field_capabilities/field_caps_response.ts

```diff
@@ -116,6 +116,8 @@ export function readFieldCapsResponse(
             }),
             {}
           ),
+          // @ts-expect-error
```

Comment:
I created a PR to address this - https://github.com/elastic/elasticsearch-js/pull/1547